build, qt: Do not require xml module for native builds in depends

hebasto opened the pull request:
This change makes Qt tools always bootstrapped.

This PR is an alternative to #19785.

Fixes partially #14648 (the complete fix is in #21591).
Fixes #18536. On ODROID-HC1 + Armbian 21.02.2 Focal:
```
$ make -C depends qt_configured
...
Configure summary:

Build type: linux-g++ (arm, CPU features: <none>)
Compiler: gcc 9.3.0
Configuration: enable_new_dtags largefile precompile_header silent release c++11 c++14 c++1z dbus reduce_exports static stl
Build options:
  Mode ................................... release
  Optimize release build for size ........ no
  Building shared libraries .............. no
  Using C standard ....................... C11
  Using C++ standard ..................... C++1z
  Using ccache ........................... no
  Using gold linker ...................... no
  Using new DTAGS ........................ yes
  Using precompiled headers .............. yes
  Using LTCG ............................. no
  Target compiler supports:
    NEON ................................. no
  Build parts ............................ libs
Qt modules and options:
...
```


Related to #21588.

#### Guix builds:
```
$ find output -type f -name *$(git rev-parse --short HEAD)*.* -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
7af4678371b212973b1738e7d41d6c22fbdc90d98c0cb576a7bad272c5172005  output/bitcoin-5a7c487a92be-aarch64-linux-gnu-debug.tar.gz
7ea3ec3b88157d39ab6716a7c9b6cc73895fd5ad370850563b5ab977698bc370  output/bitcoin-5a7c487a92be-aarch64-linux-gnu.tar.gz
67c2139008fd532c6634bb6497352a770ed1724ea9aa3ed148424770f9f7e454  output/bitcoin-5a7c487a92be-arm-linux-gnueabihf-debug.tar.gz
597ad1f2dc17a18e76f85c8898f64fbe739ad7877f0f29e29e151b273e22e8bd  output/bitcoin-5a7c487a92be-arm-linux-gnueabihf.tar.gz
1e49412a0eb8214b24da1de1a7e932c58b5672f0a571b95bed9f2d73492000e3  output/bitcoin-5a7c487a92be-osx-unsigned.dmg
daeec6577cd998acbaebfb7700ddcd7e8a4c4c344ab8e7610d35f2a8096ecf28  output/bitcoin-5a7c487a92be-osx-unsigned.tar.gz
fd47ad54cf0e098b7d219f672709ad56e36b4277734ef960efb52b8a25370222  output/bitcoin-5a7c487a92be-osx64.tar.gz
33549d0b95fe0742a4aec4deb3d76706d5a58389e796ca61bcc5532ffef6a1f1  output/bitcoin-5a7c487a92be-powerpc64-linux-gnu-debug.tar.gz
5bf5c7925d6225e8965f53118ed40da3afb5cc3d0e1747a7cb669b20f7d2020e  output/bitcoin-5a7c487a92be-powerpc64-linux-gnu.tar.gz
b331068816e483836a2018dbc433b2c973954c8f757a570629bbb7becca518fd  output/bitcoin-5a7c487a92be-powerpc64le-linux-gnu-debug.tar.gz
8ccc55288e728771550bcbf795fb1a160b750af1a72de458acdd5eaa742939c0  output/bitcoin-5a7c487a92be-powerpc64le-linux-gnu.tar.gz
6183f5bed2bf56bbc50a0e22eb4941087b8daa855c70b0730f38bc4b0d4b567a  output/bitcoin-5a7c487a92be-riscv64-linux-gnu-debug.tar.gz
3358beef89b653343b7db308da9e54e9bb456a8691b707f697596195cfbe0e1e  output/bitcoin-5a7c487a92be-riscv64-linux-gnu.tar.gz
ff5749f04527b0b3cda38b6881c801f304f1126360bd40d4ae8f397b6ebc81c3  output/bitcoin-5a7c487a92be-win-unsigned.tar.gz
d27561eca042af2b2a56e7795415c33a471d67de57bc5fab016dc8acb0c6d195  output/bitcoin-5a7c487a92be-win64-debug.zip
8d3e81d0c711ff95107bbbe68bfd9fa480f08cb8ae56e04c1f31d974eabacb6d  output/bitcoin-5a7c487a92be-win64-setup-unsigned.exe
64475777fa89691651ec2a241198e2b1dc8994762c64c80ab679607394ce3c2f  output/bitcoin-5a7c487a92be-win64.zip
ab58aa800db6572fd980c20b0d91e4c357d78c29322ca3ec4606ecdc26421b1a  output/bitcoin-5a7c487a92be-x86_64-linux-gnu-debug.tar.gz
6006c77e94e616c54f02c5d824119687abf356ebc0e2175be730812ed02576de  output/bitcoin-5a7c487a92be-x86_64-linux-gnu.tar.gz
94c787e114695563905d893b86819abe94396510b1b71047c733f6da907259ce  output/src/bitcoin-5a7c487a92be.tar.gz
```

#### Gitian builds:
- Linux
```
Generating report
d4c7386b1da2ae1f3ce12677e8577c2d08e3c07e302a3a3dfb74774bdb8aacea  bitcoin-5a7c487a92be-aarch64-linux-gnu-debug.tar.gz
b8ce16bf1927a4d22114fe82b3d5436b4a9b275af8d817c4b2c67f936b1a6b2b  bitcoin-5a7c487a92be-aarch64-linux-gnu.tar.gz
244471f8a768ae2d293f9d947650d540a54876c34ec3df4f1565053023a8d4ee  bitcoin-5a7c487a92be-arm-linux-gnueabihf-debug.tar.gz
837814ad444c306d1f16e5cc79f8e1e7f71dd2a6772eb1830a0e3eedcd204f4f  bitcoin-5a7c487a92be-arm-linux-gnueabihf.tar.gz
d22c0a0def61a649e2634a8877773a4e26f104245c4d09c35f5e3d3b855e546e  bitcoin-5a7c487a92be-powerpc64-linux-gnu-debug.tar.gz
b86712f9ac489689099f71de975eff58e83ac2359a2dfdd239cc15d041849ff0  bitcoin-5a7c487a92be-powerpc64-linux-gnu.tar.gz
49aef6378eee5a774a4a7c1bcfa5cf48a3de19754726f5f0e72064987b342f30  bitcoin-5a7c487a92be-powerpc64le-linux-gnu-debug.tar.gz
aa10bfcc573c47c841099c8b91de2fbea98f57af683a91fe89b38f41b4e68ad3  bitcoin-5a7c487a92be-powerpc64le-linux-gnu.tar.gz
4f4e309bca67ac9a1e719a54e24d84b844b10a7fff8a391915fe06b42857ecda  bitcoin-5a7c487a92be-riscv64-linux-gnu-debug.tar.gz
027f42417ac245dde776e02905b601b77477a318a965ce4dccfe76e6e0884c83  bitcoin-5a7c487a92be-riscv64-linux-gnu.tar.gz
a553d9a0b1ab6793c0a5aeec50ccf1c74a3519836f45d5ed5baa6ed5d545d1ca  bitcoin-5a7c487a92be-x86_64-linux-gnu-debug.tar.gz
c9999f5be120c45c27730bfca692cc28d1c651c33f1d77b406bc3c819944104f  bitcoin-5a7c487a92be-x86_64-linux-gnu.tar.gz
94c787e114695563905d893b86819abe94396510b1b71047c733f6da907259ce  src/bitcoin-5a7c487a92be.tar.gz
444a4630b620afac8e7818c71ddfb71c7dde1ca245fad51313bfc10477f758f0  bitcoin-core-linux-22-res.yml
Done.
```
- Windows
```
Generating report
cb12d60927fca62a0265fddd81dfd4c4ea04033b9525e55747dd56697f3ae53a  bitcoin-5a7c487a92be-win-unsigned.tar.gz
a6e21bf9f4928fdabe35148ad62927e0ce775f50cdcfaa63eb2239bc53ba9a87  bitcoin-5a7c487a92be-win64-debug.zip
580730bffc3b5537bfdcf412180e5130ef8248e3b8e071875b3c8d6b19ca7077  bitcoin-5a7c487a92be-win64-setup-unsigned.exe
66ca1e514baef8b8a37b607198583b194c11d022eed145d47b0591ed3b05e667  bitcoin-5a7c487a92be-win64.zip
94c787e114695563905d893b86819abe94396510b1b71047c733f6da907259ce  src/bitcoin-5a7c487a92be.tar.gz
e9b2689b050bebc5d65b27796ee062f39024a7218acf4ebd790790dcd2d0058f  bitcoin-core-win-22-res.yml
Done.
```
- macOS
```
Generating report
a2ad41417bac2869736ab2ae69cc9afd4afd47b0e6892e27c51c96c5605cb4e3  bitcoin-5a7c487a92be-osx-unsigned.dmg
3644d16396488b1c1b8645201b1ce45e5e7b3da921c3949808934f5313416384  bitcoin-5a7c487a92be-osx-unsigned.tar.gz
eee1e107b340c2180ec8eb2144dedbcd4d81cdc862c5608f36d5e31b2644f4bf  bitcoin-5a7c487a92be-osx64.tar.gz
94c787e114695563905d893b86819abe94396510b1b71047c733f6da907259ce  src/bitcoin-5a7c487a92be.tar.gz
38d14016fc4cf21a0adfa9251545b9efa5f5739fac5d0598a3125e24ef9f4208  bitcoin-core-osx-22-res.yml
Done.
```

hebasto committed:
build, qt: Do not require xml module for native builds in depends

This change makes Qt tools always bootstrapped.

hebasto commented:
Rebased 5a7c487a92beec11a33f6c41283c6b24abf1b6cb -> 6aaef5db59f9f43d032ee7336b2d48df92d8bb90 ([pr21589.01](https://github.com/hebasto/bitcoin/commits/pr21589.01) -> [pr21589.02](https://github.com/hebasto/bitcoin/commits/pr21589.02)) due to the conflict with #21375.

jarolrod reviewed (commented):
running `make -C depends qt_configured` on macOS `master` with #21591 patch applied (so that there is no cross-compile and it is native):

```
Build type: macx-clang (x86_64, CPU features: cx16 mmx sse sse2 sse3 ssse3 sse4.1)
Compiler: clang (Apple) 12.0.0
Configuration: sse2 aesni sse3 ssse3 sse4_1 sse4_2 avx avx2 avx512f avx512bw avx512cd avx512dq avx512er avx512ifma avx512pf avx512vbmi avx512vl f16c largefile precompile_header rdrnd shani silent x86SimdAlways release c++11 c++14 c++1z reduce_exports static stl

Qt modules and options:
  ...
  Qt Xml ................................. no
```
How do I test this? When does this come into play?

GUIX hashes for 6aaef5db59f9f43d032ee7336b2d48df92d8bb90

```
find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum

d5baeffc654fb2847b6c731a56ec0a55c39016b16aa2cec1cc1b83f3aee75db0  guix-build-6aaef5db59f9/output/aarch64-linux-gnu/bitcoin-6aaef5db59f9-aarch64-linux-gnu-debug.tar.gz
c2b36677b096e96bcfef724c9eceed559d2a63fd02d283bf10bc9679f54bceb1  guix-build-6aaef5db59f9/output/aarch64-linux-gnu/bitcoin-6aaef5db59f9-aarch64-linux-gnu.tar.gz
b33601f8ffce5db708c1acbba7cc2fe0e6fde66eaf05e3db5a6f5941467bcdcd  guix-build-6aaef5db59f9/output/arm-linux-gnueabihf/bitcoin-6aaef5db59f9-arm-linux-gnueabihf-debug.tar.gz
2aeda1fa471b8aad0a4a0d04b5f34b643b367ab5c6ecf1eeb2118c4f3a9882ad  guix-build-6aaef5db59f9/output/arm-linux-gnueabihf/bitcoin-6aaef5db59f9-arm-linux-gnueabihf.tar.gz
fea10e9a36ffea172f284f07c7224c08faff4146ea0949dc2e245aa2cf81aacc  guix-build-6aaef5db59f9/output/dist-archive/bitcoin-6aaef5db59f9.tar.gz
b8dce5a5195954ecf31d1f02eaddf10fc48dbb2d6fc8889c771b8349ae3b3fe1  guix-build-6aaef5db59f9/output/powerpc64-linux-gnu/bitcoin-6aaef5db59f9-powerpc64-linux-gnu-debug.tar.gz
f9d9af6387358bd08d94fdad3bbef806f4e3c7662ab2902c156f81524be3139d  guix-build-6aaef5db59f9/output/powerpc64-linux-gnu/bitcoin-6aaef5db59f9-powerpc64-linux-gnu.tar.gz
7aec5392b95db52001b7bbbf35c49c637f55d1cd558e6565adf329f7ca46f12f  guix-build-6aaef5db59f9/output/powerpc64le-linux-gnu/bitcoin-6aaef5db59f9-powerpc64le-linux-gnu-debug.tar.gz
9187cd7949f0c1f8c09b395de0ba3888e31060f26d0d8d79a67bf91f49af0231  guix-build-6aaef5db59f9/output/powerpc64le-linux-gnu/bitcoin-6aaef5db59f9-powerpc64le-linux-gnu.tar.gz
3062f99ff5723dd8383ba5f7d040f1300c119464a4b65cb195888e6adc343e87  guix-build-6aaef5db59f9/output/riscv64-linux-gnu/bitcoin-6aaef5db59f9-riscv64-linux-gnu-debug.tar.gz
96371d13fd00213ea08ffbe58441b2ca9a1236fa9877cec6250889ad17b0da52  guix-build-6aaef5db59f9/output/riscv64-linux-gnu/bitcoin-6aaef5db59f9-riscv64-linux-gnu.tar.gz
1506db150cd220075c8a3aa26eb55649a383812eaaf6fe7d7e81aa0c09f6c84b  guix-build-6aaef5db59f9/output/x86_64-apple-darwin18/bitcoin-6aaef5db59f9-osx-unsigned.dmg
4da01833bd4f6a534015b399a3deb0e3d4070afccda9d528c9f0257d32814b28  guix-build-6aaef5db59f9/output/x86_64-apple-darwin18/bitcoin-6aaef5db59f9-osx-unsigned.tar.gz
fbaff7ffa391f90246d87dda890b9b719143aaad2c48b5a68ff758d9dfe41d69  guix-build-6aaef5db59f9/output/x86_64-apple-darwin18/bitcoin-6aaef5db59f9-osx64.tar.gz
58e561088c18d3c26c6da7cff9390daf16d8c03baa327bc877c3f083d2f9eb16  guix-build-6aaef5db59f9/output/x86_64-linux-gnu/bitcoin-6aaef5db59f9-x86_64-linux-gnu-debug.tar.gz
bbc3d771f6d2e45e448c42aef699752fcc95b9759969a84a712d3dc89ea7d60a  guix-build-6aaef5db59f9/output/x86_64-linux-gnu/bitcoin-6aaef5db59f9-x86_64-linux-gnu.tar.gz
9e711f781257cf95e76e94ff23406dac0755256ea38e072a6c2999a2015c35f5  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win-unsigned.tar.gz
50fb6b35863aaeef59bf042c9a75427dd13039a0383e8a53ecb7e6f2b3b57521  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win64-debug.zip
2c61f1c47b771453eb99a1d2c342310b7c8796e0f51cbbb3f45768a49c9e54ea  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win64-setup-unsigned.exe
27d3bb1e045a19d87192bb35dc5be9299e58e5280f4c53fc8dd41e760c6cf054  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win64.zip
```

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
d5baeffc654fb2847b6c731a56ec0a55c39016b16aa2cec1cc1b83f3aee75db0  guix-build-6aaef5db59f9/output/aarch64-linux-gnu/bitcoin-6aaef5db59f9-aarch64-linux-gnu-debug.tar.gz
c2b36677b096e96bcfef724c9eceed559d2a63fd02d283bf10bc9679f54bceb1  guix-build-6aaef5db59f9/output/aarch64-linux-gnu/bitcoin-6aaef5db59f9-aarch64-linux-gnu.tar.gz
b33601f8ffce5db708c1acbba7cc2fe0e6fde66eaf05e3db5a6f5941467bcdcd  guix-build-6aaef5db59f9/output/arm-linux-gnueabihf/bitcoin-6aaef5db59f9-arm-linux-gnueabihf-debug.tar.gz
2aeda1fa471b8aad0a4a0d04b5f34b643b367ab5c6ecf1eeb2118c4f3a9882ad  guix-build-6aaef5db59f9/output/arm-linux-gnueabihf/bitcoin-6aaef5db59f9-arm-linux-gnueabihf.tar.gz
fea10e9a36ffea172f284f07c7224c08faff4146ea0949dc2e245aa2cf81aacc  guix-build-6aaef5db59f9/output/dist-archive/bitcoin-6aaef5db59f9.tar.gz
b8dce5a5195954ecf31d1f02eaddf10fc48dbb2d6fc8889c771b8349ae3b3fe1  guix-build-6aaef5db59f9/output/powerpc64-linux-gnu/bitcoin-6aaef5db59f9-powerpc64-linux-gnu-debug.tar.gz
f9d9af6387358bd08d94fdad3bbef806f4e3c7662ab2902c156f81524be3139d  guix-build-6aaef5db59f9/output/powerpc64-linux-gnu/bitcoin-6aaef5db59f9-powerpc64-linux-gnu.tar.gz
7aec5392b95db52001b7bbbf35c49c637f55d1cd558e6565adf329f7ca46f12f  guix-build-6aaef5db59f9/output/powerpc64le-linux-gnu/bitcoin-6aaef5db59f9-powerpc64le-linux-gnu-debug.tar.gz
9187cd7949f0c1f8c09b395de0ba3888e31060f26d0d8d79a67bf91f49af0231  guix-build-6aaef5db59f9/output/powerpc64le-linux-gnu/bitcoin-6aaef5db59f9-powerpc64le-linux-gnu.tar.gz
3062f99ff5723dd8383ba5f7d040f1300c119464a4b65cb195888e6adc343e87  guix-build-6aaef5db59f9/output/riscv64-linux-gnu/bitcoin-6aaef5db59f9-riscv64-linux-gnu-debug.tar.gz
96371d13fd00213ea08ffbe58441b2ca9a1236fa9877cec6250889ad17b0da52  guix-build-6aaef5db59f9/output/riscv64-linux-gnu/bitcoin-6aaef5db59f9-riscv64-linux-gnu.tar.gz
1506db150cd220075c8a3aa26eb55649a383812eaaf6fe7d7e81aa0c09f6c84b  guix-build-6aaef5db59f9/output/x86_64-apple-darwin18/bitcoin-6aaef5db59f9-osx-unsigned.dmg
4da01833bd4f6a534015b399a3deb0e3d4070afccda9d528c9f0257d32814b28  guix-build-6aaef5db59f9/output/x86_64-apple-darwin18/bitcoin-6aaef5db59f9-osx-unsigned.tar.gz
fbaff7ffa391f90246d87dda890b9b719143aaad2c48b5a68ff758d9dfe41d69  guix-build-6aaef5db59f9/output/x86_64-apple-darwin18/bitcoin-6aaef5db59f9-osx64.tar.gz
58e561088c18d3c26c6da7cff9390daf16d8c03baa327bc877c3f083d2f9eb16  guix-build-6aaef5db59f9/output/x86_64-linux-gnu/bitcoin-6aaef5db59f9-x86_64-linux-gnu-debug.tar.gz
bbc3d771f6d2e45e448c42aef699752fcc95b9759969a84a712d3dc89ea7d60a  guix-build-6aaef5db59f9/output/x86_64-linux-gnu/bitcoin-6aaef5db59f9-x86_64-linux-gnu.tar.gz
9e711f781257cf95e76e94ff23406dac0755256ea38e072a6c2999a2015c35f5  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win-unsigned.tar.gz
50fb6b35863aaeef59bf042c9a75427dd13039a0383e8a53ecb7e6f2b3b57521  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win64-debug.zip
2c61f1c47b771453eb99a1d2c342310b7c8796e0f51cbbb3f45768a49c9e54ea  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win64-setup-unsigned.exe
27d3bb1e045a19d87192bb35dc5be9299e58e5280f4c53fc8dd41e760c6cf054  guix-build-6aaef5db59f9/output/x86_64-w64-mingw32/bitcoin-6aaef5db59f9-win64.zip
```
#### Gitian builds:
- Linux
```
Generating report
210d7e50d2771fd48002460361196de63a30299783d54b09a2d1a5f9ff11e244  bitcoin-6aaef5db59f9-aarch64-linux-gnu-debug.tar.gz
2daaa642e54158a1a34e5bcfd746faafd4bb8fd98efabbdccea6072f0e67f729  bitcoin-6aaef5db59f9-aarch64-linux-gnu.tar.gz
159fe498e1ce9f013f3996fa4901c2a455cd4260c0a9d5cd56ce276f11b00079  bitcoin-6aaef5db59f9-arm-linux-gnueabihf-debug.tar.gz
9f864af54a02dc1565350adcd904a0ca9a48d28024afb33a67f04024067773f6  bitcoin-6aaef5db59f9-arm-linux-gnueabihf.tar.gz
3aab59087a36d4a3c94d91d6134ffff47fa283d6a85b27e57d339c450291d36a  bitcoin-6aaef5db59f9-powerpc64-linux-gnu-debug.tar.gz
62a429f2d525f1b691af2bd5a68612996176125eb6a64e6122263d3aa12a634a  bitcoin-6aaef5db59f9-powerpc64-linux-gnu.tar.gz
dac28e679f7448b9c3b87fdf54738c1426773201b66bd6303692078189481024  bitcoin-6aaef5db59f9-powerpc64le-linux-gnu-debug.tar.gz
ff0feab8362ee96d60b205fdbfc44f799ee27e09d10c22e9a78dc1adbd36bf6d  bitcoin-6aaef5db59f9-powerpc64le-linux-gnu.tar.gz
627b6b4937498e7d29f025c0a5091f64a8974d84e771ed8175c6994dd862c90e  bitcoin-6aaef5db59f9-riscv64-linux-gnu-debug.tar.gz
1e5553efbd9421f1917271710ed2b07108520fb3b74204d1c87d7efe96ca5006  bitcoin-6aaef5db59f9-riscv64-linux-gnu.tar.gz
ea52ca1bc271493b315edf551eb51f3b74e85857cecb4c61dc24e250f252753a  bitcoin-6aaef5db59f9-x86_64-linux-gnu-debug.tar.gz
8f6badc3f8da3d8f8b733f952fc9f84ab8956b0c87604508113788848ab91e02  bitcoin-6aaef5db59f9-x86_64-linux-gnu.tar.gz
fea10e9a36ffea172f284f07c7224c08faff4146ea0949dc2e245aa2cf81aacc  src/bitcoin-6aaef5db59f9.tar.gz
6f1c353c0f8993842acf1ebd68937f0769e5d18190197fae99e7c9474e2d3300  bitcoin-core-linux-22-res.yml
Done.
```
- Windows
```
Generating report
7f576630e84702bbcc91529ff437cb30d85834ba33c14706522079e65a989ec9  bitcoin-6aaef5db59f9-win-unsigned.tar.gz
d29f138e13004eb0fded2166f39714476f2d5b59125489e1532b6aefb9a3b2d9  bitcoin-6aaef5db59f9-win64-debug.zip
94a22554bdba62e6382ac8855d250170055adfe49d1b8e4ed887be866d4b35f3  bitcoin-6aaef5db59f9-win64-setup-unsigned.exe
6c1120c08f9ef13fd9f52f3d78bd5ff45e830908e3b0597913ea89e7d4f2a9f0  bitcoin-6aaef5db59f9-win64.zip
fea10e9a36ffea172f284f07c7224c08faff4146ea0949dc2e245aa2cf81aacc  src/bitcoin-6aaef5db59f9.tar.gz
ede1b6d4360dff9a4274cc54c011beda77a883f9c2b9f76230c5a45400057bad  bitcoin-core-win-22-res.yml
Done.
```
- macOS
```
Generating report
2b070917e7f9648fc3eb345a4e7514f3ee379d4e8574061b7697eb824162499c  bitcoin-6aaef5db59f9-osx-unsigned.dmg
ba887d179586bf942d2c30e9322c91bc2ba25bc77dd309f16b39f1d332f38581  bitcoin-6aaef5db59f9-osx-unsigned.tar.gz
81c5b686f62bc6277513bfb33971cd37616b30e179e75698f3c30b3342502536  bitcoin-6aaef5db59f9-osx64.tar.gz
fea10e9a36ffea172f284f07c7224c08faff4146ea0949dc2e245aa2cf81aacc  src/bitcoin-6aaef5db59f9.tar.gz
305202e0ca2bd5d73fa62dc5c254fe765eeac87c5537cd76065864d1be2b449e  bitcoin-core-osx-22-res.yml
Done.
```

hebasto commented:
> How do I test this?

E.g., compiling with depends on ARM (see #18536).

hebasto commented:
While looking at the Qt source code I found that the original issue is solved in Qt 5.15 (see https://github.com/qt/qttools/commit/9e6943370585bdc791caae6c83555b3aa5efe063).

So just waiting for bumping the Qt version in depends to 5.15.x could be an option.

hebasto commented:
Updated 6aaef5db59f9f43d032ee7336b2d48df92d8bb90 -> d3fb790093a765cf2114cb1d732e0066c54772a1 ([pr21589.02](https://github.com/hebasto/bitcoin/commits/pr21589.02) -> [pr21589.03](https://github.com/hebasto/bitcoin/commits/pr21589.03)):

- rebased on top of the recent build system changes
- add [comment](https://github.com/bitcoin/bitcoin/pull/21589#issuecomment-833381710) about ability to drop the patch after bumping Qt in depends to 5.15

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
1f85c57c738a661d6f050e1a1ba427bba863f3a579393221b8aa6c63fd782cf9  guix-build-d3fb790093a7/output/aarch64-linux-gnu/bitcoin-d3fb790093a7-aarch64-linux-gnu-debug.tar.gz
2fc8dc6aeabaaad8eb9eac301aafbc251d0d99744ab7de6ceca95fc572d52f55  guix-build-d3fb790093a7/output/aarch64-linux-gnu/bitcoin-d3fb790093a7-aarch64-linux-gnu.tar.gz
0951ce2b56c7ae3d82d2a2fb6429a53acb93216965e4e6722488be18b01971ad  guix-build-d3fb790093a7/output/arm-linux-gnueabihf/bitcoin-d3fb790093a7-arm-linux-gnueabihf-debug.tar.gz
f8d1abeee699f6b1a6278e7a28b6a251284e5636499ce4cd91e4c741d3cbd3fa  guix-build-d3fb790093a7/output/arm-linux-gnueabihf/bitcoin-d3fb790093a7-arm-linux-gnueabihf.tar.gz
b0037883dce8d3aa82d74dc4b7afc98afd2634cad030bdd2e9fd370c1955e773  guix-build-d3fb790093a7/output/dist-archive/bitcoin-d3fb790093a7.tar.gz
8f890c526f74358d48753f438f6cd66c0ef779dcba5e57bdde3cb57c0cf00c03  guix-build-d3fb790093a7/output/powerpc64-linux-gnu/bitcoin-d3fb790093a7-powerpc64-linux-gnu-debug.tar.gz
49e11b966b7ad5bf667f1fb8cd5f03aa94bae26d6ae1c5f4eab2caeaa8382a3a  guix-build-d3fb790093a7/output/powerpc64-linux-gnu/bitcoin-d3fb790093a7-powerpc64-linux-gnu.tar.gz
a81040b622956f011c0042e0b718fb7fa6a004d2556e50169c50a6ae2a22a3f7  guix-build-d3fb790093a7/output/powerpc64le-linux-gnu/bitcoin-d3fb790093a7-powerpc64le-linux-gnu-debug.tar.gz
59801e96e19ecffa60a869fb24dc9df74751e4ce308eca6678b259c19c110a24  guix-build-d3fb790093a7/output/powerpc64le-linux-gnu/bitcoin-d3fb790093a7-powerpc64le-linux-gnu.tar.gz
2c509797fcd1b25ea63dfd14d545ba7cd90d186e034ba16be9b80804d591e94d  guix-build-d3fb790093a7/output/riscv64-linux-gnu/bitcoin-d3fb790093a7-riscv64-linux-gnu-debug.tar.gz
0483352a5dc41f59f5df00c955e9ffabd842b7562a28e986faf3e0d844b18eb9  guix-build-d3fb790093a7/output/riscv64-linux-gnu/bitcoin-d3fb790093a7-riscv64-linux-gnu.tar.gz
29637c9224f22f20a8c1d44e7a0f094f82d0b1515d3c46c6608621dfc260589f  guix-build-d3fb790093a7/output/x86_64-apple-darwin18/bitcoin-d3fb790093a7-osx-unsigned.dmg
67e8612748cb6afe01b7b5e7cc6fa9e22efa013fa2c174cc333f7d84ae2ff383  guix-build-d3fb790093a7/output/x86_64-apple-darwin18/bitcoin-d3fb790093a7-osx-unsigned.tar.gz
9bf1a701e20a8e7ccb913262c2430d68919b996a0134fa1b8ba9a8b34719cd6c  guix-build-d3fb790093a7/output/x86_64-apple-darwin18/bitcoin-d3fb790093a7-osx64.tar.gz
1f10a9ec95afad1bcda9729c84577ab7af4ae60bf36c46b8cab74ccfedc78c0d  guix-build-d3fb790093a7/output/x86_64-linux-gnu/bitcoin-d3fb790093a7-x86_64-linux-gnu-debug.tar.gz
41c932d7c0f08fb4d1d4574fd20c5a3855de6b49b4b8cd71df032e98ec1a4728  guix-build-d3fb790093a7/output/x86_64-linux-gnu/bitcoin-d3fb790093a7-x86_64-linux-gnu.tar.gz
be09deda9c3ff72dc23c3cd883638ae3c508cf00be8be5a47c5e64950dd1a9ec  guix-build-d3fb790093a7/output/x86_64-w64-mingw32/bitcoin-d3fb790093a7-win-unsigned.tar.gz
dcfec4f31cd4ecf04d4ac246ec4d6bd38bb848aed3991b3f4bb1ab24f82dee7c  guix-build-d3fb790093a7/output/x86_64-w64-mingw32/bitcoin-d3fb790093a7-win64-debug.zip
dc3b59d3ecf74faf9de1791ab10b0d8068c0dc4e90e0663af439ae3000d850ab  guix-build-d3fb790093a7/output/x86_64-w64-mingw32/bitcoin-d3fb790093a7-win64-setup-unsigned.exe
5de955e360fd43b4b774be8e3da978444cb707a7110d7ba260a69c33d229251c  guix-build-d3fb790093a7/output/x86_64-w64-mingw32/bitcoin-d3fb790093a7-win64.zip
```

hebasto commented:
#### Gitian builds:
- Linux
```
Generating report
bf7e7ec4bada46ebf56b8a78c4bf4f37cd55394e6c5e725a9208d10428eaaa8b  bitcoin-d3fb790093a7-aarch64-linux-gnu-debug.tar.gz
017945c41d680b47eae23614339de6ea18315e84a5bcc667cbf2730a6dc77687  bitcoin-d3fb790093a7-aarch64-linux-gnu.tar.gz
1e1d2f882d99d8da7a909f23bc0291581cefd11f22f38706baa1a2c26d24d00b  bitcoin-d3fb790093a7-arm-linux-gnueabihf-debug.tar.gz
5fdfb9fdc039071dc88537cd167b492a4024e1b687365de9ed93b99b2d874b7a  bitcoin-d3fb790093a7-arm-linux-gnueabihf.tar.gz
2f749284c05c9112a361b91d034df5a76140dda3e745b09d7de20ef68f91a463  bitcoin-d3fb790093a7-powerpc64-linux-gnu-debug.tar.gz
e7f14aedb2c9feda511234377a284afe0f7673190e0696ee87d74e6e1114d22d  bitcoin-d3fb790093a7-powerpc64-linux-gnu.tar.gz
b111b3704f0b89c1a97d83129fca3609ed66661ff9b816c6d933fd54b825c1d4  bitcoin-d3fb790093a7-powerpc64le-linux-gnu-debug.tar.gz
45e62e1b72e613f5c577e4d5a8d803294b35a558fdc6dc8f8cf89a7d951fc12a  bitcoin-d3fb790093a7-powerpc64le-linux-gnu.tar.gz
1ad41f0099d33c7fdeed2dfd0b27e163265c194cb1b56caa6c74882174c907a8  bitcoin-d3fb790093a7-riscv64-linux-gnu-debug.tar.gz
4f26ab4df73a9881050438756be611c5584d2e027908fd7209352344d1275ffc  bitcoin-d3fb790093a7-riscv64-linux-gnu.tar.gz
f0f45f130460a0081486d0bd6a87da4f36719558c1e949deaf42403bba3e8a28  bitcoin-d3fb790093a7-x86_64-linux-gnu-debug.tar.gz
d1b4539aef4166b6dc23542322327e6f2583f2fe0954fa9470b42bd314c4dae5  bitcoin-d3fb790093a7-x86_64-linux-gnu.tar.gz
b0037883dce8d3aa82d74dc4b7afc98afd2634cad030bdd2e9fd370c1955e773  src/bitcoin-d3fb790093a7.tar.gz
8c382d4ae5ccd06605a73c40cf6c0dc89808d47ce96c76f5324963498f618bc1  bitcoin-core-linux-22-res.yml
Done.
```
- Windows
```
Generating report
8b7a9e32658c2487222c4b39b8f0dcc80c6556da229683dde3576de8c5ea52a3  bitcoin-d3fb790093a7-win-unsigned.tar.gz
8d7264dc5fe871eaee40a37cb2ef097b9f3812ff18748f2c61d29d7d446d091e  bitcoin-d3fb790093a7-win64-debug.zip
cce0d00be0670aa27e3ed342b0d006d0f4cff1cca359aa43718f21c73b6ead14  bitcoin-d3fb790093a7-win64-setup-unsigned.exe
03e1dc7854cd78d63387698b852c9c79d2c7feabbb2e7871625c96fa6047263a  bitcoin-d3fb790093a7-win64.zip
b0037883dce8d3aa82d74dc4b7afc98afd2634cad030bdd2e9fd370c1955e773  src/bitcoin-d3fb790093a7.tar.gz
310739688f8f6c51b5627df1c4b1c135b4887f4424097838b634007356d2675a  bitcoin-core-win-22-res.yml
Done.
```
- macOS
```
Generating report
e12b6da1f5ce9350f9b7237ae1a783778d4981a1b733f90146876d7392db3bb0  bitcoin-d3fb790093a7-osx-unsigned.dmg
890de901c1e0cd0a39a0cdb78fb19137206a67cf22fb0813df0e5d4c3bee79ad  bitcoin-d3fb790093a7-osx-unsigned.tar.gz
63f351f535e5fd839a4d7ae4ed9a48a83c818e60132cb32683dd5cc8137a91d9  bitcoin-d3fb790093a7-osx64.tar.gz
b0037883dce8d3aa82d74dc4b7afc98afd2634cad030bdd2e9fd370c1955e773  src/bitcoin-d3fb790093a7.tar.gz
ffde3eb281a16ba4a432c1ec8a7167b2bb5ffd6c5654defb4603a59515c1e392  bitcoin-core-osx-22-res.yml
Done.
```

fanquake commented on depends/patches/qt/force_bootstrap.patch:
```diff
@@ -0,0 +1,20 @@
+
+We should be able to drop this once we are using Qt 5.15:
+ - QTBUG-76177
+ - upstream commit: 9e6943370585bdc791caae6c83555b3aa5efe063
```
Where does this commit come from?

```bash
# git remote -v
origin	https://code.qt.io/cgit/qt/qtbase.git (fetch)
origin	https://code.qt.io/cgit/qt/qtbase.git (push)

# git branch --contains 9e6943370585bdc791caae6c83555b3aa5efe063
error: no such commit 9e6943370585bdc791caae6c83555b3aa5efe063
```

If I look for one of the actual deprecation commits i.e [82d02b7b95908dec16e41c5af1c63579729c589b](https://codereview.qt-project.org/gitweb?p=qt%2Fqtbase.git;a=commit;h=82d02b7b95908dec16e41c5af1c63579729c589b):
```bash
# git branch --contains 82d02b7b95908dec16e41c5af1c63579729c589b
  5.15
  5.15.0
  5.15.1
  5.15.2
  6.0
  6.0.0
  6.1
  dev
```

hebasto replied:
Thanks for checking!
Sorry, I don't know what was in my clipboard at that time :man_facepalming: 

Indeed, you are pointing at the right commit https://github.com/qt/qtbase/commit/82d02b7b95908dec16e41c5af1c63579729c589b

hebasto replied:
[Updated](https://github.com/bitcoin/bitcoin/pull/21589#issuecomment-839582392).

hebasto commented:
Updated d3fb790093a765cf2114cb1d732e0066c54772a1 -> 3d92cc45b30ebef4ca3af7cc02f3a7e703e81653 ([pr21589.03](https://github.com/hebasto/bitcoin/commits/pr21589.03) -> [pr21589.04](https://github.com/hebasto/bitcoin/commits/pr21589.04), [diff](https://github.com/hebasto/bitcoin/compare/pr21589.03..pr21589.04)):

- addressed @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/21589#discussion_r630662364)

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
ec191b49115658bcdac67d7c3e3fc358ca917f2bb754d01c6145d40301d0b58c  guix-build-3d92cc45b30e/output/aarch64-linux-gnu/bitcoin-3d92cc45b30e-aarch64-linux-gnu-debug.tar.gz
146de7009bab0b96612b781ea14c8e21375621785d6b09a42e875555ddfdc5c6  guix-build-3d92cc45b30e/output/aarch64-linux-gnu/bitcoin-3d92cc45b30e-aarch64-linux-gnu.tar.gz
9abeacc76dfc0e982a9dc5235caeac56587030b5194638ce756e41d44fcfdc81  guix-build-3d92cc45b30e/output/arm-linux-gnueabihf/bitcoin-3d92cc45b30e-arm-linux-gnueabihf-debug.tar.gz
d598bd68a0706c30de967640f4aa3f3ca1c619bd71f5eedf8dcd1494fb3d14c5  guix-build-3d92cc45b30e/output/arm-linux-gnueabihf/bitcoin-3d92cc45b30e-arm-linux-gnueabihf.tar.gz
8b73facb9839903e8c1ff0702de2a3fdfe185e7d2a94603348cf40a8b8c12ae3  guix-build-3d92cc45b30e/output/dist-archive/bitcoin-3d92cc45b30e.tar.gz
47cbef7ec0102e82ed619b6b0cf423c0829ed5807f4249469fa85d693ed32df0  guix-build-3d92cc45b30e/output/powerpc64-linux-gnu/bitcoin-3d92cc45b30e-powerpc64-linux-gnu-debug.tar.gz
59cb0d8c4077cc2219f62f43219f7e715d0b744c253c08aa0b636a6fab34b9a8  guix-build-3d92cc45b30e/output/powerpc64-linux-gnu/bitcoin-3d92cc45b30e-powerpc64-linux-gnu.tar.gz
8d0150c8e5e7937a8ed90779a96f3cc7f61ca85f8cc7298d49df9ce362baaf5e  guix-build-3d92cc45b30e/output/powerpc64le-linux-gnu/bitcoin-3d92cc45b30e-powerpc64le-linux-gnu-debug.tar.gz
edb8f72dc17451a51132a8a479711fdee3bdd82cbedea3d5000f660b0bcfffca  guix-build-3d92cc45b30e/output/powerpc64le-linux-gnu/bitcoin-3d92cc45b30e-powerpc64le-linux-gnu.tar.gz
8ba702ae5639275883815e1baaf771a53adb3fd48aafa20daed0be2476158ef4  guix-build-3d92cc45b30e/output/riscv64-linux-gnu/bitcoin-3d92cc45b30e-riscv64-linux-gnu-debug.tar.gz
a4f263ed5288aef20547d985ab241ab4c8a4481fc0730854809d91f52ed44a35  guix-build-3d92cc45b30e/output/riscv64-linux-gnu/bitcoin-3d92cc45b30e-riscv64-linux-gnu.tar.gz
3d33d7fcfeb181509e99cdc5fae6fb904eb12781528fa87308705f420fd19d49  guix-build-3d92cc45b30e/output/x86_64-apple-darwin18/bitcoin-3d92cc45b30e-osx-unsigned.dmg
f197ace4ff82641eec3571338c1568d849dff43c34cb14ab462a109448a4927a  guix-build-3d92cc45b30e/output/x86_64-apple-darwin18/bitcoin-3d92cc45b30e-osx-unsigned.tar.gz
89fc8d5e35a04e805cd8092f74470646ccec609d9920b5265a433e8984e9dbf0  guix-build-3d92cc45b30e/output/x86_64-apple-darwin18/bitcoin-3d92cc45b30e-osx64.tar.gz
41038d8bd3406248a32170d6aae107e9cee15c11ba19962861018c46002f91cc  guix-build-3d92cc45b30e/output/x86_64-linux-gnu/bitcoin-3d92cc45b30e-x86_64-linux-gnu-debug.tar.gz
41ca3f8c05415ead5ba2b3837a23f78144a93c7ab9d3c912a1a452524e6e2837  guix-build-3d92cc45b30e/output/x86_64-linux-gnu/bitcoin-3d92cc45b30e-x86_64-linux-gnu.tar.gz
c023b44755e5d7f2e7435782baca2d77a1e6f63fe168a2a5bd1fc0f98de70a50  guix-build-3d92cc45b30e/output/x86_64-w64-mingw32/bitcoin-3d92cc45b30e-win-unsigned.tar.gz
b3c4c2af9cf948889934fad19ae364b087aa904f386571d387dcb8a960c02a5b  guix-build-3d92cc45b30e/output/x86_64-w64-mingw32/bitcoin-3d92cc45b30e-win64-debug.zip
77fdb6ae044b8c838f9caafad9fd051f2096030d9c26eaa4f49eb7586a04a02a  guix-build-3d92cc45b30e/output/x86_64-w64-mingw32/bitcoin-3d92cc45b30e-win64-setup-unsigned.exe
278dc6216399f5db5297875c4bdd15cd1e22ad36a89239ab89d516f89b946717  guix-build-3d92cc45b30e/output/x86_64-w64-mingw32/bitcoin-3d92cc45b30e-win64.zip
```
#### Gitian builds:
- Linux
```
Generating report
9f0d7d1f8b7ff0a2c112f01830a31ecb33c3b118644c24a38e4b8f534efdd7be  bitcoin-3d92cc45b30e-aarch64-linux-gnu-debug.tar.gz
3ecb7e435ab74a371d118489a5e91e25fb2047c003b49a1f3fe242914e5972b7  bitcoin-3d92cc45b30e-aarch64-linux-gnu.tar.gz
5b7dc82ab5e1e2461e15be58a42fdedc7727a28702b29d435732ef3b95549b6f  bitcoin-3d92cc45b30e-arm-linux-gnueabihf-debug.tar.gz
f9cace86c803359aa73329891dee119ababb5a728a8018448fc26a13af3f5a8b  bitcoin-3d92cc45b30e-arm-linux-gnueabihf.tar.gz
c91c7724065f80c078281ebca718eeaf786e06d8c0b6ef423d5a901e8e8b0502  bitcoin-3d92cc45b30e-powerpc64-linux-gnu-debug.tar.gz
a1d01cee60ee8fd6b61edcc795d3ba52f5c0f709b24910fec267adb94c7b0aa1  bitcoin-3d92cc45b30e-powerpc64-linux-gnu.tar.gz
660eee90a46af6d91257dc1b444f87190b228c753a933bd5a3c879b76a0ff6ef  bitcoin-3d92cc45b30e-powerpc64le-linux-gnu-debug.tar.gz
56a5f3d293e85e40971fecd149800f65891f9d95933de6c31d0eedb23d598964  bitcoin-3d92cc45b30e-powerpc64le-linux-gnu.tar.gz
3793b94d16f36b9e5543457706efb126f76c8a1c4fa6eb6e7c4881d4d6d31e65  bitcoin-3d92cc45b30e-riscv64-linux-gnu-debug.tar.gz
df4903b11ee48d87e54e1fe2cf06367e197e6a7feb6c3ec5dc74d4f4e5c672b5  bitcoin-3d92cc45b30e-riscv64-linux-gnu.tar.gz
f33ff3d167cfcda749f76fb52b4cd1cae331349ea0140d5d175c0713f29260de  bitcoin-3d92cc45b30e-x86_64-linux-gnu-debug.tar.gz
e3b9e5234abd96b5e385b647d43fb40023b508d07d84dc99f2ac479054966914  bitcoin-3d92cc45b30e-x86_64-linux-gnu.tar.gz
8b73facb9839903e8c1ff0702de2a3fdfe185e7d2a94603348cf40a8b8c12ae3  src/bitcoin-3d92cc45b30e.tar.gz
34f0fed7e3267444c533176281d7a1d6ad31c5680d3e6a6d86d18cf08acc87c0  bitcoin-core-linux-22-res.yml
Done.
```
- Windows
```
Generating report
2fdf5eefce7641dd3a58627ae9e42ab6282d60c945e919b295c6cdc48a2727e8  bitcoin-3d92cc45b30e-win-unsigned.tar.gz
e7e67c490b71cbf1d240e564fd31416afcffc8f273979aa92f6382690f240843  bitcoin-3d92cc45b30e-win64-debug.zip
e57e7d332f5b22ffda922db55a7f5acf483d63dfefa08ff9900dd045cb2e123e  bitcoin-3d92cc45b30e-win64-setup-unsigned.exe
64a5d5930abafc72dba7f2148a02d3f677ec859d179ba18351e3b99b8cfe9780  bitcoin-3d92cc45b30e-win64.zip
8b73facb9839903e8c1ff0702de2a3fdfe185e7d2a94603348cf40a8b8c12ae3  src/bitcoin-3d92cc45b30e.tar.gz
c523221b39930bad496964871fe1e053e4f53db4ed7e3197ad80dbefdaa92116  bitcoin-core-win-22-res.yml
Done.
```
- macOS
```
Generating report
483578ba800677efe8be010000dc3196c80d579e624abf23def1b7f6cb81e7a3  bitcoin-3d92cc45b30e-osx-unsigned.dmg
1ea9dc8b45522d1bd20889f9fe7f8fbda25c722dbae63f5cf6e44f95fede5884  bitcoin-3d92cc45b30e-osx-unsigned.tar.gz
7e94485ae4368c24ffb99890d59921c0c07e1f8edb2255f986a37f1345d4b530  bitcoin-3d92cc45b30e-osx64.tar.gz
8b73facb9839903e8c1ff0702de2a3fdfe185e7d2a94603348cf40a8b8c12ae3  src/bitcoin-3d92cc45b30e.tar.gz
e56593600e02e2b6e200a289074a26a618bfeef575fb2079e3638a2e91e8d91b  bitcoin-core-osx-22-res.yml
Done.
```

hebasto commented:
Rebased 3d92cc45b30ebef4ca3af7cc02f3a7e703e81653 -> e14d89aa36015c5951097364c0d20f3127097413 ([pr21589.04](https://github.com/hebasto/bitcoin/commits/pr21589.04) -> [pr21589.05](https://github.com/hebasto/bitcoin/commits/pr21589.05)) due to the conflict with #22054.

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
4fc0cfc776800b3060d75be6cb188ba95d093e5def1a8c4a5c58715d861a798f  guix-build-e14d89aa3601/output/aarch64-linux-gnu/bitcoin-e14d89aa3601-aarch64-linux-gnu-debug.tar.gz
9f006b7a5b2d3db64af26f9acb2099b57f2df564beae6abe037b87d1dcf4afbf  guix-build-e14d89aa3601/output/aarch64-linux-gnu/bitcoin-e14d89aa3601-aarch64-linux-gnu.tar.gz
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/aarch64-linux-gnu/inputs.SHA256SUMS
72a859427be139fe6f14815fbd94005f60c2422c830f5d3b47cc128e9d5f36ee  guix-build-e14d89aa3601/output/arm-linux-gnueabihf/bitcoin-e14d89aa3601-arm-linux-gnueabihf-debug.tar.gz
29a617e3812b7edd836741dd5f5154047b787af9ab60b4257a44be644df5b780  guix-build-e14d89aa3601/output/arm-linux-gnueabihf/bitcoin-e14d89aa3601-arm-linux-gnueabihf.tar.gz
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/arm-linux-gnueabihf/inputs.SHA256SUMS
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  guix-build-e14d89aa3601/output/dist-archive/SKIPATTEST.TAG
49e073e3b335bc42f92378a354e1eec03f190f1babd1e334fe532133ba0b6980  guix-build-e14d89aa3601/output/dist-archive/bitcoin-e14d89aa3601.tar.gz
c571a9bebf9027269e2e3133c646cd3afddeccecdf6f84c402efd7aa9a6b85c8  guix-build-e14d89aa3601/output/powerpc64-linux-gnu/bitcoin-e14d89aa3601-powerpc64-linux-gnu-debug.tar.gz
3b0154e205047be7ab6b2b1dc78442c3d7bb521ba49abd54bbad2fbf9b54113b  guix-build-e14d89aa3601/output/powerpc64-linux-gnu/bitcoin-e14d89aa3601-powerpc64-linux-gnu.tar.gz
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/powerpc64-linux-gnu/inputs.SHA256SUMS
c21613b5add1ee6e6c170ea623149681926317950f1f0c61ce0a76933f818ec6  guix-build-e14d89aa3601/output/powerpc64le-linux-gnu/bitcoin-e14d89aa3601-powerpc64le-linux-gnu-debug.tar.gz
26e20fb3313b2e1e8685b03d161a0d3548220a27c7148829f57b4c56b49e7775  guix-build-e14d89aa3601/output/powerpc64le-linux-gnu/bitcoin-e14d89aa3601-powerpc64le-linux-gnu.tar.gz
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/powerpc64le-linux-gnu/inputs.SHA256SUMS
6b3b89a30e00bf42f67d0cb5d9a65a51f58a7cb7badee33b234939f46092e33e  guix-build-e14d89aa3601/output/riscv64-linux-gnu/bitcoin-e14d89aa3601-riscv64-linux-gnu-debug.tar.gz
87456d918e0cec3ddacbad5b902b9e4143357f7ccaa482a44a47e4d37ae958ed  guix-build-e14d89aa3601/output/riscv64-linux-gnu/bitcoin-e14d89aa3601-riscv64-linux-gnu.tar.gz
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/riscv64-linux-gnu/inputs.SHA256SUMS
5342a4a297bcd0d1b7607bc328ac04c9a916e1adb67c5e5113958dfb1a4186a6  guix-build-e14d89aa3601/output/x86_64-apple-darwin18/bitcoin-e14d89aa3601-osx-unsigned.dmg
20efb4f207f4a4be0c21c8709a4b617aade2047eaf0529c094a7ef0fb0c57ad9  guix-build-e14d89aa3601/output/x86_64-apple-darwin18/bitcoin-e14d89aa3601-osx-unsigned.tar.gz
b8b01e68706ce140f4e665baa5321d23c50ca058be527685080a71e32d8aab56  guix-build-e14d89aa3601/output/x86_64-apple-darwin18/bitcoin-e14d89aa3601-osx64.tar.gz
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/x86_64-apple-darwin18/inputs.SHA256SUMS
41b8fabbf08209b32eb634251e160115cbce4179e687adae4bfd47dd185981ba  guix-build-e14d89aa3601/output/x86_64-linux-gnu/bitcoin-e14d89aa3601-x86_64-linux-gnu-debug.tar.gz
128de416002d771068bfc0529497936a46eec145d26a52fda4f07a1bdca9fbc0  guix-build-e14d89aa3601/output/x86_64-linux-gnu/bitcoin-e14d89aa3601-x86_64-linux-gnu.tar.gz
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/x86_64-linux-gnu/inputs.SHA256SUMS
32f56621abd2a0b31ac8652aff9d28e8cf45a728b2b6f5b7a2c24f76da6fda1b  guix-build-e14d89aa3601/output/x86_64-w64-mingw32/bitcoin-e14d89aa3601-win-unsigned.tar.gz
2d1335efe630b19d5c16d5903dbbd0da044dea1e4dd881b07cbd94807f22964f  guix-build-e14d89aa3601/output/x86_64-w64-mingw32/bitcoin-e14d89aa3601-win64-debug.zip
0e19d0e5c509b4168b553cb75f3b3ab783dad5de9fd75e9fa01d0a653eea912c  guix-build-e14d89aa3601/output/x86_64-w64-mingw32/bitcoin-e14d89aa3601-win64-setup-unsigned.exe
f525564f343a02d337782067ecb6a7e482d2ba439329b272d4aba82250e5903d  guix-build-e14d89aa3601/output/x86_64-w64-mingw32/bitcoin-e14d89aa3601-win64.zip
45416202e20ba0b720d37367b5076653a252b1efebb95519c959efb26e075384  guix-build-e14d89aa3601/output/x86_64-w64-mingw32/inputs.SHA256SUMS
```

hebasto commented:
Could the "22.0" milestone be assigned to this PR?

fanquake commented:
> Could the "22.0" milestone be assigned to this PR?

NACK. After thinking about this more, I don't want to go down this path of continually hacking up Qt. I've got some changes that I'll draft PR shortly, which should remove the need for this, and some of your other open PRs, while fixing the same issues they are trying to fix.

fanquake commented:
For now, I'm going to close this in favour of #22142. If, for some reason, we can't get that change done fairly shortly, I'll reopen this PR.
